easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-foss-2018b-Python-3.7.0.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['a328d63d476b3653f5a25b5f7971e87a15cdf8860ab0729d4b1157ba988b8d0b']
+
+dependencies = [
+    ('Python', '3.7.0'),
+    ('netCDF', '4.6.1'),
+    ('expat', '2.2.5', '-GCCcore-7.3.0', True),
+    ('GEOS', '3.6.2', '-Python-%(pyver)s'),
+    ('SQLite', '3.24.0'), # GCCcore
+    ('libxml2', '2.9.8'), # GCCcore
+    ('libpng', '1.6.34'), # GCCcore
+    ('libjpeg-turbo', '2.0.0', '-GCCcore-7.3.0', True),
+    ('JasPer', '2.0.14'),
+    ('LibTIFF', '4.0.9', '-GCCcore-7.3.0', True),
+    ('zlib', '1.2.11'),
+    ('cURL', '7.60.0', '-GCCcore-7.3.0', True),
+    ('PCRE', '8.41'),
+    ('PROJ', '5.0.0'),
```

Comment:
`libgeotiff` dependency is missing (see `GDAL-2.2.3-foss-2018a-Python-3.6.4.eb`)